website/content/en/highlights/2021-10-05-0-17-upgrade-guide.md

```diff
@@ -0,0 +1,29 @@
+---
+date: "2021-10-05"
+title: "0.17 Upgrade Guide"
+description: "An upgrade guide that addresses breaking changes in 0.17.0"
+authors: ["tobz"]
+pr_numbers: []
+release: "0.17.0"
+hide_on_release_notes: false
+badges:
+  type: breaking change
+---
+
+Vector's 0.17.0 release includes three **breaking changes**:
```

Comment:
```suggestion
Vector's 0.17.0 release includes one **breaking change**:
```

Reply:
@tobz's is just a fortune teller, we _will_ make 3 breaking changes.